src/Raven.Client/Documents/Operations/ETL/Queue/AwsSqsConnectionSettings.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Client.Documents.Operations.ETL.Queue;
+
+public sealed class AwsSqsConnectionSettings
+{
+    public Basic Basic { get; set; }
+
+    public bool Passwordless { get; set; }
+
+    public bool UseEmulator { get; set; }
```

Comment:
Shouldn't it be `internal`? It's for testing purposes only and it requires a custom env variable `RAVEN_AWS_SQS_EMULATOR_URL`

Reply:
If it is internal how can use it in tests?

Reply:
We use `InternalsVisibleTo` attribute so it will be accessible in our tests